jobs/post_int2lm.py

```diff
@@ -91,7 +90,7 @@ def main(start_time, hstart, hstop, cfg, model_cfg):
 
     # Meteo spinup simulation with tracer recycling
     if hasattr(cfg, 'spinup') and \
-    hasattr(cfg, 'post_int2lm_species_spinup') and not cfg.first_one:
+    cfg.post_int2lm.get('species_spinup') is not None and not cfg.first_one:
```

Comment:
Just checking: (1) does this correspond to `cfg.post_int2lm.species_spinup` or to `cfg.post_int2lm_species_spinup` (as in the next line), and (2) the logic changed, because in the past it might not have had this attribute at all and now it assumes this attribute is definitely present -- do I see that correctly, and is that on purpose?

Reply:
The `get()` method returns `None` if the key doesn't exist. For that reason, the logic remains essentially the same.
See https://www.w3schools.com/python/ref_dictionary_get.asp

Reply:
Great! What about the first question then [=the variable `cfg.post_int2lm_species_spinup` in line 94 should then be replaced with `cfg.post_int2lm.get('species_spinup')`, no?].

Reply:
Ah sorry I didn't see line 94 in the comment preview. Good catch, I will correctly replace the variable with the next commit.

Reply:
Solved in https://github.com/C2SM/processing-chain/pull/36/commits/5e10925e2e0460247d4bbf6d428bbf00640dafae